src/js/App/App.js

```diff
@@ -1,13 +1,12 @@
 import React, { Component } from 'react';
 
+import './App.scss';
 import NasaTable from '../NasaTable';
 import Header from '../Header';
 
 import sorts from '../sorts/sorts';
 import getInfoFromAPI from '../api/api';
 
-import './App.scss';
```

Comment:
please place all the imports in the order 'libs - empty line - project imports - empty line -  style imports'